rackcity/api/serializers/asset_serializers.py

```diff
@@ -102,6 +102,8 @@ class ChassisSerializer(serializers.ModelSerializer):
     """
 
     rack = RackSerializer()
+    model = ITModelSerializer()
+    blades = serializers.SerializerMethodField()
```

Comment:
adding model so that a blade know's it chassis' display color, and blades that a blade knows the other blades in it's chassis ( for the diagram)

Reply:
hell yeah